pos_order_zipcode/models/pos_order.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Moka Tourisme (https://www.mokatourisme.fr).
+# @author Pierre Verkest <pierreverkest84@gmail.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    zipcode = fields.Char("Zip")
```

Comment:
The rest of odoo name the zipcode ``zip``. (partner / bank / company).

Reply:
I thought about it, `zipcode` is used in `base_address_city` module and I'm not english I've feeling zipcode is less ambiguis than zip ?

Reply:
.... and also because [zip](https://docs.python.org/3/library/functions.html#zip) is a python built-in function !

Reply:
Well. 
regarding naming, ``zip_code`` (``zipcode``) is indeed better. 

if you execute ``rgrep "zipcode.*field" ./ --include=*.py``, you'll see that most of the fields are ``zip``. But indeed there a lost ``zipcode`` in base_adress_city``. (and also a ``zip_code`` in ``lunch`` V15 module).

My preference is to follow the (pseudo-)convention of odoo and wait they change.

Not a blocking point for me.